Source/WebCore/css/MediaQueryEvaluator.cpp

```diff
@@ -481,6 +481,33 @@ static bool dynamicRangeEvaluate(CSSValue* value, const CSSToLengthConversionDat
     }
 }
 
+static bool scanEvaluate(CSSValue* value, const CSSToLengthConversionData&, Frame& frame, MediaFeaturePrefix)
+{
+    auto view = frame.view();
+    if (!view)
+        return false;
+
+    /* With Media Queries Level 4, the "tv" media type is deprecated
+     * and the "scan" feature applies to all media types.
+     * We are currently supporting Media Queries Level 3,
+     * thus the check against "tv".
+     */
+    auto mediaType = frame.view()->mediaType();
+    static constexpr auto tv = ASCIILiteral::fromLiteralUnsafe("tv");
```

Comment:
The `static` here avoid copying the data onto the stack, but considering that the data is only ~3bytes, I'm not sure it's actually faster.

Reply:
I think the check below can just be `equalLettersIgnoringASCIICase(mediaType, "tv"_s)`, similar to `evaluateResolution`.

(cc @cdumez for string best-practices)

Reply:
@pxlcoder is right. Just use "tv"_s to get an ASCIILiteral. If you're using something with "unsafe" in the name, you're probably doing it wrong :)

Reply:
Fixed ✅